src/main/resources/Ideas/IdeasSheet.xml

```diff
@@ -356,9 +372,48 @@
   color: $theme.linkColor;
   cursor: pointer;
   text-decoration: underline;
+}
+
+.recent {
+  line-height: 30px;
+}
+
+.recent .xvote-pro, .recent .xvote-con {
+  margin: .1em .3em .5em 0;
+  width: 15px;
+  height: 15px;
+  line-height: 15px;
+  font-size: 15px;
+  padding: 1px;
+  border-radius: 0;
+  font-size: .8em;
+}
+
+.recent .noitems {
+  font-size: inherit;
```

Comment:
the 'noitems' behavior is to put a smaller size. If you don't like the font-size added, than don't use the '.noitems' class, but make a custom class that adds only the color.